app/controllers/account_activations_controller.rb

```diff
@@ -0,0 +1,14 @@
+class AccountActivationsController < ApplicationController
+  def edit
+    user = User.find_by email: params[:email]
```

Comment:
tách hẵng code handle user ra 1 method mới nhé nhớ handle cả user not found, sau đó dùng action filter để call cho edit nhé